### ISSUE TITLE
`default_locale` issue

### ISSUE THREAD

**Issue by danschultzer** [kipcole9/cldr#38]
I went from 1.3.2 to master with the following config:

```elixir
config :ex_cldr,
  default_locale: "en",
  locales: ["en"],
  gettext: MyApp.Gettext
```

But I got this error:

```
 ** (FunctionClauseError) no function clause matching in Keyword.get/3

     The following arguments were given to Keyword.get/3:
     
         # 1
         nil
     
         # 2
         :default_locale
     
         # 3
         nil
     
     Attempted function clauses (showing 1 out of 1):
     
         def get(keywords, key, default) when is_list(keywords) and is_atom(key)
     
     stacktrace:
       (elixir) lib/keyword.ex:179: Keyword.get/3
       (ex_cldr) lib/cldr/config/config.ex:245: Cldr.Config.gettext_locales/0
       (ex_cldr) lib/cldr.ex:369: Cldr.known_gettext_locale_name?/1
       (ex_cldr) lib/cldr.ex:448: Cldr.known_gettext_locale_name/1
       (ex_cldr) lib/cldr/locale.ex:328: Cldr.Locale.first_match/2
       (ex_cldr) lib/cldr/locale.ex:317: Cldr.Locale.gettext_locale_name/1
       (ex_cldr) lib/cldr/locale.ex:299: Cldr.Locale.set_gettext_locale_name/1
       (ex_cldr) lib/cldr/locale.ex:249: Cldr.Locale.canonical_language_tag/1
       (ex_cldr) lib/cldr.ex:536: Cldr.validate_locale/1
```

To fix it, I had to add the following:

```elixir
config :my_app, MyApp.Gettext, default_locale: "en"
```

Is this expected behavior? If it is, then it should have a proper warning.

**Comment by kipcole9** [comment]
Fixed on master.  Thanks for the report.

**Comment by kipcole9** [comment]
Published on hex as version 1.4.0-rc.1
